stock_picking_customer_ref/tests/test_stock_picking_customer_ref.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2015 AvanzOSC - Alfredo de la Fuente
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestStockPickingCustomerRef(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestStockPickingCustomerRef, cls).setUpClass()
+        cls.product = cls.env['product.product'].create({
+            'name': 'Test stuff',
+        })
+        cls.partner = cls.env['res.partner'].create({
+            'name': 'Mr. Odoo',
+        })
+        cls.sale_order = cls.env['sale.order'].create({
```

Comment:
@bizzappdev some object `cls.env['sale.order']`  use multiple time, So use as like a global variable.

Reply:
@nikul-serpentcs IMO it is okay as it has been used only once outside the setup method.

Reply:
@bizzappdev Yes, I know but IMO if you use as a global variable that good for a feature.